setup.py

```diff
@@ -12,6 +12,7 @@
 
 
 class build_ext(_build_ext):
+
```

Comment:
Strange place for an empty line, especially with a docstring following it

Reply:
PEP 257

Reply:
PEP257 doesn't specify a blank line before the docstring, it specifies a blank line after.

Reply:
See Error Code [D203](https://pep257.readthedocs.io/en/latest/error_codes.html) 

> 1 blank line required before class docstring

But we can leave it as it is.

Reply:
D203 is deprecated and should be disabled if your editor doesn’t do this already:

See:

https://opendev.org/openstack/keystoneauth/commit/cf520d9ca45dfbb09ca9bab90fa3d615bc950326

Reply:
Yeah, I've seen that there has been a discussion. That's why I reverted the change already.